library/alloc/src/fmt.rs

```diff
@@ -273,6 +273,37 @@
 //! Hello, `     123` has 3 right-aligned characters
 //! ```
 //!
+//! For the floating-point type, it employs the
+//! [roundTiesToEven](https://en.m.wikipedia.org/wiki/Rounding#Rounding_half_to_even) convention:
```

Comment:
What does the name "roundTiesToEven" refer to exactly? I did not find that anywhere in the link.